tests/phpunit/QueryPrinterRegistryTestCase.php

```diff
@@ -2,8 +2,10 @@
 
 namespace SMW\Tests;
 
+use SMW\Query\ResultPrinters\ResultPrinter;
+
 /**
- * Base class for \SMW\Query\ResultPrinters\ResultPrinter tests.
+ * Base class for ResultPrinter tests.
```

Comment:
Not related to your change, but this comment is noise and could also be removed

Reply:
Thanks, done.